WORKSPACE

```diff
@@ -286,3 +288,11 @@ maven_jar(
   name = "org_mockito_mockito_all",
   artifact = "org.mockito:mockito-all:1.10.19",
 )
+
+git_repository(
+  name = "io_bazel_rules_pex",
+  remote = "https://github.com/benley/bazel_rules_pex.git",
```

Comment:
Better home: yes, I hope so.  I haven't directly requested a github project from bazelbuild, but moving it out of my personal account does seem appropriate before merging this change.

Internal CI: it would work equally well to get these rules from a .zip or .tgz archive using a [http_archive rule](http://www.bazel.io/docs/be/workspace.html#http_archive), or even from a path on disk using [local_repository](http://www.bazel.io/docs/be/workspace.html#local_repository) if that turns out to be easier.  In theory we could publish a pypi project containing the bazel rules, though it would be a bit awkward.  I'm not very familiar with Maven - can it be used to deliver arbitrary files?  If so, I imagine that could be made to work as well.

If none of that is practical, we could just vendor things back into this repo rather than fetching it externally as part of the build.